lib/pbench/test/unit/server/test_datasets_list.py

```diff
@@ -85,13 +85,9 @@ def get_results(self, name_list: List[str], query: JSON, server_config) -> JSON:
             Paginated JSON object containing list of dataset values
         """
         list: List[JSON] = []
-        page_limit = query.get(
-            "page_limit", int(server_config.server_config.get("page_limit"))
-        )
-        page_no = query.get("page_no", 1)
-        paginated_name_list = name_list[
-            (page_no - 1) * page_limit : page_no * page_limit
-        ]
+        limit = query.get("limit", len(name_list) + 1)
+        offset = query.get("offset", 0)
+        paginated_name_list = name_list[offset : (offset + 1) * limit]
```

Comment:
I may be confused, but shouldn't that be
```
        paginated_name_list = name_list[offset : offset + limit]
```
That is, shouldn't the size of the slice be the same regardless of the offset?

Reply:
Ah! you are right it should be `+` not `*`